### ISSUE TITLE
Documenting buggy behaviour while dynamic config

### ISSUE THREAD

**Issue by ghassanmas** [overhangio/tutor-mfe#86]
## Problematic configs 

 1. `ACCESS_TOKEN_COOKIE_NAME`
          When this doesn't exists: `ACCESS_TOKEN_COOKIE_NAME:"edx-jwt-cookie-header-payload"`
          
          js
          lib.js:151 Uncaught Error: getLocale called before configuring i18n. Call configure with messages first.
              at E (lib.js:151:11)
              at p (ErrorPage.jsx:26:40)
              at Va (react-dom.production.min.js:153:146)
              at bu (react-dom.production.min.js:261:496)
              at ds (react-dom.production.min.js:246:265)
              at As (react-dom.production.min.js:246:194)
              at ns (react-dom.production.min.js:239:172)
              at Ju (react-dom.production.min.js:230:137)
              at Ls (react-dom.production.min.js:281:43)
              at react-dom.production.min.js:284:301
          
This occurs on frontend-app-learning when being authenticated, it will not occur if you browsing a course as anonymous. 
          
 2. `STUDIO_BASE_URL` I found that course authoring doesn't take this key into consideration, though it makes call to mfe config, but nonetheless it doesn't affect. 
 
 ## Solution Proposal
 1. The solution of one can be ensure that value is should be set at build time and if operator want to change they need to rebuild the image
 2. Create a proxy role where wet STUDIO_BASE_URL as `cms-api`  and then create a proxy role simliar to the one for config api that proxy  `cms-api` to => `STUDIO_BASE_URL`. One thing I am not sure about this approach is that how it would work with k8s, but in any case it uses the same principle of the how we proxy the config api. 
 
On a more general point, I think it would better to just create two proxy role, for the LMS and CMS such that always set a predifiend token for of both and then add proxy to lms/cms, instead of just adding proxy for dynamic config uri we set it at the sub/domain level.

   3. [Edit] I found the cause while it's picking it, and suggest a fix at https://github.com/openedx/frontend-app-course-authoring/pull/389, though I still see good value in the above.

**Comment by arbrandes** [comment]
@ghassanmas, I'm closing this in favor of opening new issues for specific problems after #69 is merged.
